financial-templates-lib/logger/ConsoleTransport.js

```diff
@@ -18,10 +18,9 @@ const customFormat = combine(
     // Winston does not properly log Error objects like console.error() does, so this formatter will search for the Error object
     // in the "error" property of "info", and add the error stack to the log.
     // Discussion at https://github.com/winstonjs/winston/issues/1338.
-    if (error && error.stack) {
```

Comment:
Removing this check for `error.stack` means that `ConsoleTransport` is necessary because by the time `error`'s are sent to this module they already have had their `stack` stripped out by `Logger.js: errorStackTracerFormatter()`